RadixWallet/Features/SplashFeature/Splash.swift

```diff
@@ -37,6 +37,25 @@ public struct Splash: Sendable, FeatureReducer {
 		case completed(Profile)
 	}
 
+	public struct Destination: DestinationReducer {
+		public enum State: Sendable, Hashable {
+			case passcodeCheckFailed(AlertState<Action.PasscodeCheckFailedAlert>)
+		}
+
+		public enum Action: Sendable, Equatable {
+			case passcodeCheckFailed(PasscodeCheckFailedAlert)
+
+			public enum PasscodeCheckFailedAlert: Sendable, Equatable {
+				case retryButtonTapped
+				case openSettingsButtonTapped
+			}
+		}
+
+		public var body: some ReducerOf<Self> {
+			EmptyReducer()
```

Comment:
Do we really need an empty body? I think not? But perhaps including it makes type checking faster?